data-es6.js

```diff
@@ -2914,6 +2914,30 @@ exports.tests = [
   },
 },
 {
+  name: 'prototype of bound functions',
+  category: 'functions',
+  significance: 'small',
+  link: 'https://people.mozilla.org/~jorendorff/es6-draft.html#sec-boundfunctioncreate',
+  exec: function () {/*
+    function getProtoBound(proto) {
+      var f = Object.setPrototypeOf(function() { }, proto)
```

Comment:
fwiw, this test will fail in any engine without `Object.setPrototypeOf` - is it worth creating a helper polyfill like the one in es6-shim, so that browsers with `__proto__` support but not `setPrototypeOf` can still pass this test (when testing shims, for example)?